packages/ui/components/footer/Update.tsx

```diff
@@ -0,0 +1,26 @@
+import { Heading, Flex, Text } from "@chakra-ui/react";
+import React from "react";
+
+export interface UpdateProps {
+    title: string
+    description: string
+}
+
+export const Update = ({ title, description } : UpdateProps) => {
```

Comment:
Is there a better way to name this component other than "Update"? This name might be abit confusing 🤔